src/instructlab/eval/mmlu.py

```diff
@@ -30,10 +36,49 @@ def run(self) -> tuple:
             overall_score       MMLU score for the overall model evaluation
             individual_scores   Individual MMLU score for each task
         """
-        individual_scores: dict[str, float] = {}
-        overall_score: float = 0.0
+        #TODO: make this a parameter for class?
+        os.environ['TOKENIZERS_PARALLELISM'] = 'true'
+
+        individual_scores: dict = {}
+        agg_score: float = 0.0
+        model_args = "pretrained=" + self.model_path + ",dtype=" + self.model_dtype
+
+        mmlu_output = simple_evaluate(
+            model="hf",
+            model_args=model_args,
+            tasks=self.tasks,
+            num_fewshot=self.few_shots,
+            batch_size=self.batch_size
+        )
+
+        results = mmlu_output["results"]
+
+        for task in self.tasks:
+            mmlu_res = results[task]
+            agg_score += float(mmlu_res['acc,none'])
+            individual_scores[task] = {}
+            individual_scores[task]['score'] = float(mmlu_res['acc,none'])
+            individual_scores[task]['stderr'] = float(mmlu_res['acc_stderr,none'])
+
+        overall_score = float(agg_score/len(self.tasks))
         return overall_score, individual_scores
+    
+
+############# Testing Code Follows ##############
```

Comment:
Could we add this to a separate file like @danmcp did in #9?

That way we can run it via the work that will be done in #12

Reply:
We should absolutely move this into a separate file! That being said I think we should do it in a follow up PR. We should just remove this `Testing Code Follows` section from the PR.

Reply:
That's fine with me! Will give me a chance to setup runners anyway

Reply:
Removed the testing code! Will re-add into a separate file in a followup PR like Ali suggested.

Reply:
Added issue here: https://github.com/instructlab/eval/issues/15